src/Presence/rythm.json

```
@@ -0,0 +1,22 @@
+{
+    "rythm.openDashboard": {
+        "message": "Open Dashboard",
+        "description": "Button for opening the web dashboard."
+    },
+    "rythm.inviteBot": {
+        "message": "Invite Bot",
+        "description": "Button for inviting the bot."
+    },
+    "rythm.djs": {
+        "message": "DJs:",
+        "description": "The amount of users with DJ permissions will be listed after the \":\""
+    },
```

```suggestion
    "rythm.djs": {
        "message": "DJs: {0}",
        "description": "{0} Will be replaced with the amount of users with DJ permissions."
    },
```
Or if you have the count on another line:
```suggestion
    "rythm.djs": {
        "message": "DJs: {0}{1}",
        "description": "{0} is an enter and is required. {1} Will be replaced with the amount of users with DJ permissions."
    },
```